src/main/kotlin/org/koitharu/kotatsu/parsers/site/en/LineWebtoonsParser.kt

```diff
@@ -0,0 +1,426 @@
+package org.koitharu.kotatsu.parsers.site.en
+
+import okhttp3.Headers
+import org.json.JSONObject
+import org.koitharu.kotatsu.parsers.MangaLoaderContext
+import org.koitharu.kotatsu.parsers.MangaParser
+import org.koitharu.kotatsu.parsers.MangaSourceParser
+import org.koitharu.kotatsu.parsers.config.ConfigKey
+import org.koitharu.kotatsu.parsers.exception.ParseException
+import org.koitharu.kotatsu.parsers.model.*
+import org.koitharu.kotatsu.parsers.util.*
+import org.koitharu.kotatsu.parsers.util.json.mapJSON
+import org.koitharu.kotatsu.parsers.util.json.mapJSONIndexed
+import org.koitharu.kotatsu.parsers.util.json.mapJSONToSet
+import org.koitharu.kotatsu.parsers.util.json.toJSONList
+import java.net.URI
+import java.net.URLEncoder
+import java.util.*
+import javax.crypto.Mac
+import javax.crypto.spec.SecretKeySpec
+
+internal abstract class LineWebtoonsParser(context: MangaLoaderContext, source: MangaSource) : MangaParser(context, source)  {
+	private val signer = WebtoonsUrlSigner("gUtPzJFZch4ZyAGviiyH94P99lQ3pFdRTwpJWDlSGFfwgpr6ses5ALOxWHOIT7R1")
+
+	override val configKeyDomain
+		get() = ConfigKey.Domain("webtoons.com")
+	private val configKeyApiDomain
+		get() = ConfigKey.Domain("global.apis.naver.com")
+	private val configKeyStaticDomain
+		get() = ConfigKey.Domain("webtoon-phinf.pstatic.net")
```

Comment:
Multiple `ConfigKey.Domain` will not work because this is actually the same key.

I guess currently the best solution will be a hardocde these domains (as a constants ofc)